app/migrations/1-initial.js

```diff
@@ -0,0 +1,34 @@
+var fs = require('fs');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return new Promise( (resolve, reject) => {
+      fs.readFile(__dirname + '/../../resources/initial.sql', function(err, data){
```

Comment:
Preferable to use an absolute path - the project's root dir is defined in process.env.CWD - will update this after merge.